app/controllers/SignUpController.scala

```diff
@@ -58,6 +58,7 @@ class SignUpController @Inject() (
             lastName = Some(data.lastName),
             fullName = Some(data.firstName + " " + data.lastName),
             email = Some(data.email),
+            role = Some("Student"),
```

Comment:
You are right. This was updated later. However the Naming convention for Constants in scala is UpperCamelCas and the same is being followed.